website/content/commands/operator/raft.mdx

```diff
@@ -104,3 +104,29 @@ Usage: `consul operator raft remove-peer -address="IP:port"`
 - `-id` - ID of the server to remove.
 
 The return code will indicate success or failure.
+
+## transfer-leader
+
+Corresponding HTTP API Endpoint: [\[POST\] /v1/operator/raft/transfer-leader](/api-docs/operator/raft#transfer-leader)
+
+This command transfer the Consul leader to another server. If an `id` is provided, raft leadership will be transferred
+to the server with that id.
+
+[`consul members`](/commands/members) command, it is preferable to
+clean up by simply running
+[`consul force-leave`](/commands/force-leave)
+instead of this command.
+
```

Comment:
Is this supposed to be here?

```suggestion
[`consul members`](/commands/members) command, it is preferable to
clean up by simply running
[`consul force-leave`](/commands/force-leave)
instead of this command.

```

Reply:
copy-pasta!